### ISSUE TITLE
Incorrect Comment syntax highlighting for block comments with string note

### ISSUE THREAD

**Issue by amerikan** [squ1b3r/Djaneiro#124]
Block comments allows for an optional string note. See https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#comment

```
<p>Rendered text with {{ pub_date|date:"c" }}</p>
{% comment "Optional note" %}    <------------ EXAMPLE HERE
    <p>Commented out text with {{ create_date|date:"c" }}</p>
{% endcomment %}
```

Without a string note, the comment syntax highlighted appears a muted gray as expected.

![Screen Shot 2020-08-03 at 4 13 06 PM](https://user-images.githubusercontent.com/179356/89235994-3ba14900-d5a4-11ea-9177-6af498298c21.png)

However, when a note is added it no longer appears as if it was a comment:

![Screen Shot 2020-08-03 at 4 14 28 PM](https://user-images.githubusercontent.com/179356/89236089-7c00c700-d5a4-11ea-8d13-82977050749e.png)

Expected:
Should be also be a muted gray color.

Basically the following two formats should have grayed out appearance.

```
{% comment %} {% endcomment %}
```

```
{% comment "a string" %} {% endcomment %}
```

I'm not sure if this is by design and if I just have to change something in my settings.

**Comment by amerikan** [comment]
Without looking to deep into the source code, my initial guess would be that we would need to add a regex here: https://github.com/squ1b3r/Djaneiro/blob/master/Syntaxes/HTML%20(Django).tmLanguage ?

**Comment by willstott101** [comment]
I've been meaning to get round to opening a PR for merging: https://github.com/willstott101/django-sublime-syntax

I'd think it'd be easier to fix this syntax error in the .sublime-syntax format which my repo there uses. The newer syntax also fixes: #102

**Comment by amerikan** [comment]
@willstott101 that would be great. I think that's my only complaint about the package, other than that it works perfect for my use cases.

**Comment by scuml** [comment]
This has been resolved.
